<launch>
  <arg name="gui" default="false"/>

  <param name="robot_description" command="xacro --inorder '$(find hb_viz)/urdf/hummingbird.urdf.xacro'" />
  <param name="use_gui" value="$(arg gui)"/>

  <node name="joint_state_publisher" pkg="hb_viz" type="joint_state_publisher.py" output="screen"/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find hb_viz)/rviz/hummingbird.rviz" required="true" output="screen"/>
</launch>
